### ISSUE TITLE
MaxRetry with {} Queue Option Argument throwing Nil Error

### ISSUE THREAD

**Issue by anuonifade** [ruby-amqp/kicks#21]
When Using a MaxRetry handler option for Queues and Queue Option argument is an empty hash, it throws an error.

```
name = "sample-max-retry"
opts[:queue_options] = {:durable=>true, :auto_delete=>false, :exclusive=>false, :arguments=>{}}
Sneakers::Handlers::Maxretry.configure_queue(
  name,
  opts
)
```
The line https://github.com/ruby-amqp/kicks/blob/185b51c7d6759d432eb005ddf199eff3886381b4/lib/sneakers/handlers/maxretry.rb#L87 will throw an error.

`opts[:queue_options][:arguments] ? ` will return true and trying to `inject` and empty hash will raise an error.

Changing the line
```
opt_args = opts[:queue_options][:arguments] ? opts[:queue_options][:arguments].inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo} : {}
```
To
```
opt_args = opts[:queue_options][:arguments].blank? ?  {} :  opts[:queue_options][:arguments].inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
```
Should fix the issue.

**Comment by michaelklishin** [comment]
Feel free to submit a PR.

**Comment by michaelklishin** [comment]
``` ruby
opt_args = opts[:queue_options][:arguments].blank? ?  {} :  opts[:queue_options][:arguments].inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
```

should probably be

``` ruby
opt_args = if opts[:queue_options][:arguments].blank?
             {}
           else
             opts[:queue_options][:arguments].transform_keys(&:to_sym)
           end
```

**Comment by anuonifade** [comment]
Yeah, that looks ruby like. I will raise the PR

**Comment by michaelklishin** [comment]
@anuonifade Bunny and Kicks require 2.5 and arguably should require 2.7. Should have done it a couple of years ago, in fact.

**Comment by michaelklishin** [comment]
Addressed in #22 by @anuonifade and @texpert.
